packages/amplication-client/src/Entity/EntityFieldForm.tsx

```diff
@@ -129,38 +131,45 @@ const EntityFieldForm = ({
             <DisplayNameField
               name="displayName"
               label="Display Name"
-              disabled={isDisabled}
+              disabled={isDisabled || isSystemData}
               required
             />
             <NameField name="name" disabled={isDisabled} required />
```

Comment:
so, for system data types we cannot change the dispaly name, but we can change the field name?

Reply:
These are all disabled on type ID in production
![image](https://user-images.githubusercontent.com/39680385/202394205-3fec13ca-e937-498b-adc5-6e2dee11d301.png)